x-pack/plugins/fleet/server/routes/enrollment_api_key/index.ts

```diff
@@ -21,43 +21,51 @@ import {
   postEnrollmentApiKeyHandler,
 } from './handler';
 
-export const registerRoutes = (routers: { superuser: FleetRouter; fleetSetup: FleetRouter }) => {
-  routers.fleetSetup.get(
+export const registerRoutes = (router: FleetAuthzRouter) => {
+  router.get(
     {
       path: ENROLLMENT_API_KEY_ROUTES.INFO_PATTERN,
       validate: GetOneEnrollmentAPIKeyRequestSchema,
-      // Disable this tag and the automatic RBAC support until elastic/fleet-server access is removed in 8.0
-      // Required to allow elastic/fleet-server to access this API.
-      // options: { tags: [`access:${PLUGIN_ID}-read`] },
+      fleetAllowFleetSetupPrivilege: true,
+      fleetAuthz: {
+        fleet: { readEnrollmentTokens: true },
+      },
```

Comment:
Similar as above, do we need the `fleetAllowFleetSetupPrivilege: true` flag if we already have `readEnrollmentTokens` which is set in `calculateAuthz`? Same for the other `readEnrollmentTokens` endpoints below.